### DISCUSSION TITLE
Fedramp compliant?

### DISCUSSION THREAD

**Discussion by chrisbigelow** [supabase/supabase#19800] [General]
Is Supabase FEDRAMP compliant? Or does it adhere to NIST 800-53 standards?

**Comment by 24601** [comment]
@chrisbigelow - we are investigating the same/similar right now. If you are still interested in question, would love to have a chat directly. Not a sales person, not trying to sell you anything, just sounds like you're facing similar questions as us and maybe we could share what we've learned, but I am reticent to discuss our use case and needs publicly for reasons I am sure you understand. 

I sent you a note from my work email on your GH listed email.

**Reply by vishnumurali03** [reply]
Also investigating the same thing. Would love to chat

**Comment by mdvertola** [comment]
Bumping this
